maintainers/maintainer-list.nix

```diff
@@ -5269,6 +5269,12 @@
     githubId = 31556469;
     name = "Prem Netsuwan";
   };
+  Grubben = {
```

Comment:
Split this file into its own commit, with message `maintainer-list: add grubben`

https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits

Reply:
oh I see, I'm going to try

Reply:
Done!